### DISCUSSION TITLE
Moving a tree of pages in 2.x

### DISCUSSION THREAD

**Discussion by mzdimal** [requarks/wiki#7431] [Help / Questions]
We have a tree of hundreds of pages. It doesn't seem possible to move an entire branch of pages to a new location. However, is there a backend way to do this?

**Comment by fnafox** [comment]
I think the easiest way to accomplish this would be using storage sync. I advise creating a dbdump before proceeding so you can restore the wiki if needed.
1. Navigate to Admin > Storage
2. Create a content dump to storage target of your choice
3. Disable storage sync
4. Reorganize pages in your page dump
5. Make a full import from the storage target (this will overwrite all wiki content!)
